pkg/users/account-details.js

```diff
@@ -343,3 +334,104 @@ export function AccountDetails({ accounts, groups, shadow, current_user, user })
         </Page>
     );
 }
+
+export const AccountGroupsSelect = ({ name, groups, setError }) => {
+    const usedGroups = groups.filter(group => group.userlist.includes(name));
+    const primaryGroup = groups.find(group => group.userlistPrimary.includes(name));
+    const primaryGroupName = primaryGroup ? primaryGroup.name : undefined;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17996-20221206-122041-ea567d53-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)